approval/repository.go

```diff
@@ -53,13 +56,38 @@ func (r *repository) ListApprovals(conditions *domain.ListApprovalsFilter) ([]*d
 	}
 	db = db.Where(`"Appeal"."status" != ?`, domain.AppealStatusCanceled)
 
-	db = db.Clauses(clause.OrderBy{
-		Expression: clause.Expr{
-			SQL:                `ARRAY_POSITION(ARRAY[?], "approvals"."status"), "updated_at" desc`,
-			Vars:               []interface{}{ApprovalStatusDefaultSort},
-			WithoutParentheses: true,
-		},
-	})
+	if conditions.OrderBy != nil {
+		var orderByClauses []string
```

Comment:
Should we move this code inside the if block to another function for readibility?

Reply:
updated